src/styles/blocks/_product.scss

```diff
@@ -0,0 +1,85 @@
+.product {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: $color-main;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    margin-top: 40px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__code {
+    margin-top: 5px;
+
+    font-size: 10px;
+    color: $color-secondary;
+  }
+
+  &__rating-container, &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__rating-container {
+    margin-top: 18px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    color: $color-main-accent;
+  }
+
+  &__price-container {
+    margin-top: 23px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    color: $color-secondary;
+  }
+
+  &__price {
+    font-size: 16px;
+    font-weight: 700;
+    color: $color-main-accent;
+  }
+
+  &__buy-btn {
+    display: block;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    text-align: center;
+    border: 1px solid $color-blue-accent;
```

Comment:
you should create transparent border and change its color in :hover

Reply:
fixed